base/docs/basedocs.jl

```diff
@@ -1938,7 +1978,9 @@ isdefined
 """
     Vector{T}(undef, n)
 
-Construct an uninitialized [`Vector{T}`](@ref) of length `n`. See [`undef`](@ref).
+Construct an uninitialized [`Vector{T}`](@ref) of length `n`.
+
+See also [`undef`](@ref), [`similar`](@ref).
```

Comment:
I think this change would be better suited for the primary docstring of ```Vector{T} <: AbstractVector{T}``` (for see also `similar`, `fill`, and `zeros`)

Reply:
and `empty`

Reply:
Done in https://github.com/JuliaLang/julia/pull/38606/commits/5f4284b63873e3942a7f79a27bb705417cce5f69

Reply:
I think `empty` was not previously in docs. Added in https://github.com/JuliaLang/julia/pull/38606/commits/924f0fc13e18d3426ee07eab340216923d563f65 but I am not very sure that `doc/src/base/arrays.md ` is the right place, since it applies to many collections.